todo_app/app.py

```diff
@@ -1,11 +1,20 @@
-from flask import Flask
+from crypt import methods
```

Comment:
I think this is an unused import, and actually errors on Windows, so let's get rid of this first line!